modules/installation-guide/partials/proc_importing-untrusted-tls-certificates.adoc

```diff
@@ -22,121 +24,82 @@ endif::[]
+For a {prod-short} link:https://helm.sh/[Helm Chart] deployment only one config map supported. It is reccomended to add the certificates at installation time.
 
-[subs="+attributes,+quotes",options="nowrap",role=white-space-pre]
-----
-$ {orch-cli} get checluster {prod-checluster} -n __<{project-context}-namespace-name>__ -o jsonpath={.spec.server.serverTrustStoreConfigMapName}
-----
-====
-ifeval::["{project-context}" == "che"]
-+
-====
-On instances of {prod-short} deployed with the {prod-short} link:https://helm.sh/[Helm Chart] deployment, retrieve the name of the ConfigMap by reading the `global.tls.serverTrustStoreConfigMapName` property from the Helm Chart:
+=== At installation time
 
+The config map should be pre-created in the {prod} installation namespace.
+Ensure you override the `global.tls.serverTrustStoreConfigMapName` Helm Chart property with the name of the config map when installing the {prod-short} Helm Chart. For this you should add the following arguments to the Helm command line:
 [subs="+quotes",options="nowrap",role=white-space-pre]
 ----
-$ helm get values che --all --output json | jq -r '.global.tls.serverTrustStoreConfigMapName'
+--set global.tls.serverTrustStoreConfigMapName=custom-certs
 ----
-====
-endif::[]
-+
-[NOTE]
-====
-If the existing installation did not define any name for the ConfigMap, just use `ca-certs`.
-====
 
-.Procedure
-
-include::example$snip_importing-untrusted-tls-certificates_1.adoc[]
-
-. Add the required TLS certificates in the ConfigMap:
-+
-[subs="+attributes,+quotes"]
+Or, in case of `{prod-cli}` usage, provide a Helm patch `--helm-patch-yaml` parameter that points to an yaml file with:
+[subs="+quotes",options="nowrap",role=white-space-pre]
 ----
-$ {orch-cli} create configmap __<config-map-name>__ --from-file=__<certificate-file-path>__ -n=__<{project-context}-namespace-name>__ -o yaml --dry-run | {orch-cli} apply -f -
+global:
+  tls:
+    enabled: true
+    serverTrustStoreConfigMapName: custom-certs
 ----
-+
-To apply more than one certificate, add another `--from-file=_<certificate-file-path>_` option to the above command.
 
-. Configure the {prod-short} installation to use the ConfigMap:
-+
-====
-For a {prod-short} link:https://docs.openshift.com/container-platform/latest/operators/olm-what-operators-are.html[Operators] deployment:
-
-. Edit the `spec.server.ServerTrustStoreConfigMapName` `CheCluster` Custom Resource property to match the name of the ConfigMap:
-+
-[subs="+attributes,+quotes",options="nowrap",role=white-space-pre]
-----
-$ {orch-cli} patch checluster {prod-checluster} -n __<{project-context}-namespace-name>__ --type=json -p '[{"op": "replace", "path": "/spec/server/serverTrustStoreConfigMapName", "value": "<config-map-name>"}]'
-----
-+
-====
-ifeval::["{project-context}" == "che"]
-+
-====
-For a {prod-short} link:https://helm.sh/[Helm Chart] deployment: 
+=== On already running installations
+For a {prod-short} link:https://helm.sh/[Helm Chart] deployment:
```

Comment:
Remove this since we already mentioned that above.

Reply:
In the step below we can remove mentions about `ingressDomain` since we don't update anything. I believe it is copy/paste.

Reply:
ok